### ISSUE TITLE
Archive CardZoom (card back) image is broken

### ISSUE THREAD

**Issue by romanogit** [keyteki/keyteki#212]
The archive CardZoom image is broken. Should we have zoom for archive pile anyway since it is just the back of the card?

I am checking it while developing https://github.com/keyteki/keyteki/issues/147

**Comment by cryogen** [comment]
There are some known oddities with opponents archive that I've been meaning to fix.  Thanks for raising them!

First as you say, opponents archives shouldn't be mouse overable, unless it's revealed (like opponents hand for example)

Second is that your cards show face up in the opponents archive, which is also wrong

**Comment by romanogit** [comment]
Relates to #187

**Comment by romanogit** [comment]
> First as you say, opponents archives shouldn't be mouse overable, unless it's revealed (like opponents hand for example)

@cryogen : Do you happen to know a card that would reveal opponent's archive. Also, shouldn't that behave the same was as hand reveal, through a message in the game message panel?

> Second is that your cards show face up in the opponents archive, which is also wrong
I was able to recreate this one. It also shows an yellow border around the archive pile.

Me looking at my archive pile with 2 enemy creatures and 1 friend creature:
![image](https://user-images.githubusercontent.com/2104075/62539372-75bdfa80-b82b-11e9-9ce1-93d919fc4c04.png)

My opponent looking at my archive pile with his two creatures and one mine:
![image](https://user-images.githubusercontent.com/2104075/62539454-9dad5e00-b82b-11e9-9363-1a051436aa1d.png)

Basically, opponent should never be able to click on my archive (see first question above) and my archive should always show me all creatures, regardless is mine or opponent's.

I will work on this one. Feel free to assign it to me.

**Comment by romanogit** [comment]
This issue was partially fixed by https://github.com/keyteki/keyteki/pull/424
